models/user.js

```diff
@@ -0,0 +1,42 @@
+const Sequelize = require("sequelize");
+
+const sequelize = require("../util/database");
+const Role = require("../models/role");
+const user = sequelize.define(
+  "user",
+  {
+    username: {
+      type: Sequelize.STRING,
```

Comment:
username data type is wrong @amrit56789 
check https://sequelize.org/docs/v7/other-topics/other-data-types/ and implement whatever is best for the fileds